### ISSUE TITLE
The book does not build

### ISSUE THREAD

**Issue by wdomitrz** [pbiecek/xai_stories#39]
* OS: Ubuntu 20.04 LTS
* R version: 3.6.3 (2020-02-29) -- "Holding the Windsock"
* pandoc version: 
```
$ pandoc --version                                                                                              (git)-[master] 
pandoc 2.9.2
Compiled with pandoc-types 1.20, texmath 0.12.0.1, skylighting 0.8.3.2
Default user data directory: /home/witalis/.local/share/pandoc or /home/witalis/.pandoc
Copyright (C) 2006-2019 John MacFarlane
Web:  https://pandoc.org
This is free software; see the source for copying conditions.
There is no warranty, not even for merchantability or fitness
for a particular purpose.
```
* `sessionInfo` (after the `devtools::install_dev_deps()` command):
```
> sessionInfo()
R version 3.6.3 (2020-02-29)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.9.0
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.9.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=pl_PL.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=pl_PL.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=pl_PL.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=pl_PL.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] ps_1.3.3          fansi_0.4.1       prettyunits_1.1.1 rprojroot_1.3-2  
 [5] withr_2.2.0       digest_0.6.25     crayon_1.3.4      assertthat_0.2.1 
 [9] R6_2.4.1          backports_1.1.7   magrittr_1.5      rlang_0.4.6      
[13] cli_2.0.2         fs_1.4.1          remotes_2.1.1     testthat_2.3.2   
[17] callr_3.4.3       ellipsis_0.3.1    desc_1.2.0        devtools_2.3.0   
[21] tools_3.6.3       glue_1.4.1        pkgload_1.1.0     compiler_3.6.3   
[25] processx_3.4.2    pkgbuild_1.0.8    sessioninfo_1.1.1 memoise_1.1.0    
[29] usethis_1.6.1
```

### Step 2: Install dependencies

```{r}
devtools::install_dev_deps()
```

### Step 3: Render the book (R commands)

```{r}
bookdown::render_book('./', 'bookdown::gitbook')
```

## Result
```
Quitting from lines 2197-2199 (book.Rmd) 
Error in knitr::include_graphics("images/09_recency.PNG") : 
  Cannot find the file(s): "images/09_recency.PNG"
```

## Possible steps to fix

After creating a `images/09_recency.PNG` file, with the same steps:

```
Quitting from lines 2197-2199 (book.Rmd) 
Error in knitr::include_graphics("images/09_history.PNG") : 
  Cannot find the file(s): "images/09_history.PNG"
```

After creating a `images/09_history.PNG` file, with the same steps:
```
Quitting from lines 2220-2222 (book.Rmd) 
Error in knitr::include_graphics("images/09_uplift_train.png") : 
  Cannot find the file(s): "images/09_uplift_train.png"
```

After creating a `images/09_uplift_train.png` file, with the same steps:
```
Quitting from lines 2220-2222 (book.Rmd) 
Error in knitr::include_graphics("images/09_uplift_test.png") : 
  Cannot find the file(s): "images/09_uplift_test.png"
```

After creating a `images/09_uplift_test.png` file, with the same steps:
```
Quitting from lines 2280-2282 (book.Rmd) 
Error in knitr::include_graphics("images/09_shap_min_uplift.png") : 
  Cannot find the file(s): "images/09_shap_min_uplift.png"
```

After creating a `images/09_shap_min_uplift.png` file, with the same steps:
```
Quitting from lines 2280-2282 (book.Rmd) 
Error in knitr::include_graphics("images/09_shap_max_uplift.png") : 
  Cannot find the file(s): "images/09_shap_max_uplift.png"
```

After creating a `images/09_shap_max_uplift.png` file, with the same steps the build is succesfill (with some warnings)
```
pandoc-citeproc: reference 4groups_chart not found
pandoc-citeproc: reference pyuplift not found
pandoc-citeproc: reference pdp not found
pandoc-citeproc: reference shap_dependence_plot not found
pandoc-citeproc: reference shap_dependence_plot_explained not found
```

## Additional context

Additionally files:
```
images/09_history.png
images/09_recency.png
images/09_shap_max_uplift.PNG
images/09_shap_min_uplift.PNG
images/09_uplift_test.PNG
images/09_uplift_train.PNG
```
are not used in the build.

## Possible fix

Based on this information I had, I submitted a PR, namely:
```
        renamed:    images/09_history.png -> images/09_history.PNG
        renamed:    images/09_recency.png -> images/09_recency.PNG
        renamed:    images/09_shap_max_uplift.PNG -> images/09_shap_max_uplift.png
        renamed:    images/09_shap_min_uplift.PNG -> images/09_shap_min_uplift.png
        renamed:    images/09_uplift_test.PNG -> images/09_uplift_test.png
        renamed:    images/09_uplift_train.PNG -> images/09_uplift_train.png
```
Unfortunately the PR was criticized by the creators of the `09_*.{PNG,png}` files.

I would be grateful for fixing this problem.

**Comment by wdomitrz** [comment]
After today's merges the book still does not compile (the same system), but produces partially different errors (the problem is with `images/{08,09}_*.{png,PNG}` files).

**Comment by pbiecek** [comment]
Thanks, the problem is fixed with debbdf7b1b94f16a0259b74a5b97fed115037473

**Comment by wdomitrz** [comment]
> Thanks, the problem is fixed with [debbdf7](https://github.com/pbiecek/xai_stories/commit/debbdf7b1b94f16a0259b74a5b97fed115037473)
